src/main/resources/interface-specification.yml

```diff
@@ -714,6 +714,45 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  '/institutions/insert/{externalId}':
+    post:
+      tags:
+        - process
+      summary: Create an institution using external institution id without fetching data from user-registry
+      description: Create an institution using external institution id without fetching data from user-registry
+      operationId: createInstitutionRaw
+      parameters:
+        - name: externalId
+          in: path
+          description: The externalId of the institution
+          required: true
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InstitutionSeed'
+      responses:
+        '201':
```

Comment:
If you want to retrieve the object created 200 is the right choice.
201 is used together with the Location header.